src/networking.c

```diff
@@ -2055,19 +2230,198 @@ static void writeToReplica(client *c) {
     c->nwritten = totwritten;
 }
 
+/* Bulk string reply requires 3 iov entries -
+ * length prefix ($<length>\r\n), string (<data>) and suffix (\r\n) */
+#define NUM_OF_IOV_PER_BULK_STR 3
+/* Bulk string prefix max size */
+#define BULK_STR_LEN_PREFIX_MAX_SIZE (LONG_STR_SIZE + 3)
```

Comment:
will it not be clearer to write:
```suggestion
#define BULK_STR_LEN_PREFIX_MAX_SIZE (LONG_STR_SIZE + NUM_OF_IOV_PER_BULK_STR)
```

Reply:
`#define BULK_STR_LEN_PREFIX_MAX_SIZE (LONG_STR_SIZE + NUM_OF_IOV_PER_BULK_STR)` is not correct.
+3 is because length prefix format is `$<length>\r\n `and not related to 3 iovs per bulk str.
Updated comment to this one:
```
/* Bulk string prefix max size (long + $ + \r\n) */
#define BULK_STR_LEN_PREFIX_MAX_SIZE (LONG_STR_SIZE + 3)
```